import.sql

```diff
@@ -130,14 +130,16 @@ CREATE TABLE IF NOT EXISTS `npwd_twitter_reports`
 CREATE TABLE IF NOT EXISTS `npwd_messages`
 (
     `id`              int(11)      NOT NULL AUTO_INCREMENT,
-    `message`         varchar(512) NOT NULL,
+    `message`         varchar(512) DEFAULT '',
     `user_identifier` varchar(48)  NOT NULL,
     `conversation_id` varchar(512) NOT NULL,
     `isRead`          tinyint(4)   NOT NULL DEFAULT 0,
     `createdAt`       timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP(),
     `updatedAt`       timestamp    NOT NULL DEFAULT CURRENT_TIMESTAMP() ON UPDATE CURRENT_TIMESTAMP,
     `visible`         tinyint(4)   NOT NULL DEFAULT 1,
     `author`          varchar(255) NOT NULL,
+    `is_embed`        tinyint(4)   NOT NULL default 0,
```

Comment:
Same thing here imo. People probably don’t wanna wipe their database after feats. Least with this we can make a simply querry just to insert these 2.